support-frontend/assets/helpers/page/page.js

```diff
@@ -122,24 +122,21 @@ function storeEnhancer(thunk: boolean) {
     : undefined;
 
 }
+
 /* eslint-enable no-underscore-dangle */
 
 // Initialises the page.
-function init<S, A>(
+function initRedux<S, A>(
   pageReducer?: CommonState => Reducer<S, A> | null,
   thunk?: boolean = false,
```

Comment:
not part of your changes, but I had to google "thunk"!
![image](https://user-images.githubusercontent.com/7304387/130080509-bfa3959b-6fb0-4a16-bb54-601874fe857e.png)